finagle-core/src/test/scala/com/twitter/finagle/util/LoadServiceTest.scala

```diff
@@ -94,6 +95,12 @@ class LoadServiceTest extends FunSuite with MockitoSugar {
     executor.shutdown()
   }
 
+  test("LoadService should ignore packages according to ignoredPaths GlobalFlag") {
```

Comment:
I tried to run a test with the requirements you specified. For some reason, even though the addition of the package "com.twitter.finagle.util" to the list of ignoredPackages used in the browseUri it still seemed to pick up the LoadServiceRandomInterfaceImpl with the LoadService. Perhaps i am missing something here :)